### PR TITLE
docs: add ClientTrafficPolicy user guide

### PR DESCRIPTION
**Pull request by cnvergence**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes https://github.com/envoyproxy/gateway/issues/2032

### CODE REVIEW

File: site/content/en/latest/user/client-traffic-policy.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+
+## Introduction
+
+The [ClientTrafficPolicy][] API allows system administrators to configure
+the behavior for how the Envoy Proxy server behaves with downstream clients.
+
+## Motivation
+
+This API was added as a new policy attachment resource that can be applied to Gateway resources and it is meant to hold settings for configuring behavior of the connection between the downstream client and Envoy Proxy listener. It the counterpart to the [BackendTrafficPolicy][] API resource.
```
Comment (zhaohuabing) [inline site/content/en/latest/user/client-traffic-policy.md:15]:
```suggestion
It's the counterpart to the [BackendTrafficPolicy][] API resource.
```

Comment (zhaohuabing) [inline site/content/en/latest/user/client-traffic-policy.md:15]:
```suggestion
This API was added as a new policy attachment resource that can be applied to Gateway resources and it is meant to hold settings for configuring behavior of the connection between the downstream client and Envoy Proxy listener. It is the counterpart to the [BackendTrafficPolicy][] API resource.
```

**Review by zhaohuabing — COMMENTED** [review]
Thanks, just a small nit.

File: site/content/en/latest/user/client-traffic-policy.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  targetRef:
+    group: gateway.networking.k8s.io
+    kind: Gateway
+    name: eg
+    namespace: default
+  enableProxyProtocol: true
+EOF
+```
+
+## Support TCP keepalive for downstream client
```
Comment (shawnh2) [inline site/content/en/latest/user/client-traffic-policy.md:43]: should this be a three-level title?

File: site/content/en/latest/user/client-traffic-policy.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+This API was added as a new policy attachment resource that can be applied to Gateway resources and it is meant to hold settings for configuring behavior of the connection between the downstream client and Envoy Proxy listener. It the counterpart to the [BackendTrafficPolicy][] API resource.
+
+## Quickstart
+
+### Prerequisites
+
+* Follow the steps from the [Quickstart](../quickstart) guide to install Envoy Gateway and the example manifest.
+Before proceeding, you should be able to query the example backend using HTTP.
+
+### Enable Proxy Protocol for downstream client
```
Comment (shawnh2) [inline site/content/en/latest/user/client-traffic-policy.md:24]: do we need to state what will happen if user apply these configs briefly?

**Review by zhaohuabing — APPROVED** [review]

**Review by zirain — COMMENTED** [review]
lgtm @arkodg PTAL

**Comment by arkodg** [comment]
we will also need to highlight the feature, using a client call https://serverfault.com/questions/305388/how-to-test-keep-alive-is-working-on-client-end/305404#305404

**Comment by arkodg** [comment]
and `curl --haproxy-protocol` for proxy protocol

**Comment by cnvergence** [comment]
Sure, I will add everything what I have tried to test this feature

**Comment by cnvergence** [comment]
proxy protocol added, no issues here

**Review by shawnh2 — APPROVED** [review]
thanks, lgtm

**Review by zirain — APPROVED** [review]

**Pull request merged**